post_office/management/commands/load_email_template.py

```diff
@@ -0,0 +1,32 @@
+from django.core.management.base import BaseCommand
+from django.template import loader
+
+from ...models import EmailTemplate
+from ...logutils import setup_loghandlers
+
+logger = setup_loghandlers()
+
+
+def _get_template(template_name):
+    """Find the template and return the raw template file content"""
+
+    template = loader.get_template(template_name)
```

Comment:
It throws an exception:
```
$ python manage.py load_email_template foo

Traceback (most recent call last):
  File "manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/home/yuri/virtualenvs/inbli-core/lib/python3.4/site-packages/django/core/management/__init__.py", line 353, in execute_from_command_line
    utility.execute()
  File "/home/yuri/virtualenvs/inbli-core/lib/python3.4/site-packages/django/core/management/__init__.py", line 345, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/yuri/virtualenvs/inbli-core/lib/python3.4/site-packages/django/core/management/base.py", line 348, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/yuri/virtualenvs/inbli-core/lib/python3.4/site-packages/raven/contrib/django/management/__init__.py", line 41, in new_execute
    return original_func(self, *args, **kwargs)
  File "/home/yuri/virtualenvs/inbli-core/lib/python3.4/site-packages/django/core/management/base.py", line 399, in execute
    output = self.handle(*args, **options)
  File "/home/yuri/repos/django-post_office/post_office/management/commands/load_email_template.py", line 29, in handle
    'subject': _get_template(template_name + '/subject.txt'),
  File "/home/yuri/repos/django-post_office/post_office/management/commands/load_email_template.py", line 13, in _get_template
    template = loader.get_template(template_name)
  File "/home/yuri/virtualenvs/inbli-core/lib/python3.4/site-packages/django/template/loader.py", line 43, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: foo/subject.txt
```
Maybe a try-except with a more descriptive message will be better.

The bigger problem is that I can't get it to work on older Django versions because of changes to templates, so I can get the template file, but not the file path or the unprocessed text of the template...

Maybe better to drop the the template loaders, and instead to add a  `POST_OFFICE_TEMPLATES_PATH` setting and just get the files from there...